airflow-postgres-app-services/azuredeploy.json

```diff
@@ -0,0 +1,263 @@
+        "pgServerName": "[concat('Microsoft.DBforPostgreSQL/servers/', variables('serverName'))]"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/sites",
+            "name": "[parameters('airflowWebAppName')]",
+            "apiVersion": "2019-08-01",
+            "kind": "app,linux,container",
+            "location": "[parameters('location')]",
+            "dependsOn": [
+                "[variables('servicePlan')]",
+                "[variables('databaseName')]"
+            ],
+            "comments": "Airflow lauched based on the latest Docker image of apache/airflow.",
+            "properties": {
+                "siteConfig": {
+                    "linuxFxVersion": "DOCKER|puckel/docker-airflow:latest",
+                    "requestTracingEnabled": true,
+                    "httpLoggingEnabled": true,
+                    "detailedErrorLoggingEnabled": true,
+                    "autoHealEnabled": true
+                },
+                "name": "[parameters('airflowWebAppName')]",
+                "httpsOnly": true,
+                "serverFarmId": "[variables('servicePlan')]"
+            },
+            "resources": [
+                {
+                    "name": "appsettings",
+                    "type": "config",
+                    "apiVersion": "2019-08-01",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.Web/sites', parameters('airflowWebAppName'))]"
+                    ],
+                    "tags": {
+                        "displayName": "Airflow App Settings"
+                    },
+                    "comments": "This are the generated settings for the Airflow Web App",
+                    "properties": {
+                        "AIRFLOW__CORE__SQL_ALCHEMY_CONN": "[concat('postgresql://', variables('AirflowUserName'),':',parameters('postgrePassword'),'@',reference(resourceId('Microsoft.DBforPostgreSQL/servers',variables('serverName'))).fullyQualifiedDomainName,':5432/',variables('databaseName'))]",
+                        "AIRFLOW__CORE__SQL_ALCHEMY_POOL_SIZE": 20,
+                        "AIRFLOW__CORE__LOAD_EXAMPLES": "true",
+                        "AIRFLOW__CORE__EXECUTOR": "LocalExecutor",
+                        "WEBSITES_ENABLE_APP_SERVICE_STORAGE": "true"
+                    }
+                }
+            ]
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "kind": "linux",
+            "location": "[parameters('location')]",
+            "name": "[variables('servicePlan')]",
+            "properties": {
+                "name": "[variables('servicePlan')]",
+                "workerSizeId": "1",
+                "reserved": true,
+                "numberOfWorkers": "1"
+            },
+            "sku": {
+                "Tier": "[parameters('servicePlanTier')]",
+                "Name": "[parameters('servicePlanSku')]"
+            },
+            "type": "Microsoft.Web/serverfarms"
```

Comment:
move this to the top of the resource definition

Reply:
Updated.